### DISCUSSION TITLE
alist启动不了，日志如下

### DISCUSSION THREAD

**Discussion by ytb1079014809** [AlistGo/alist#2830] [General]
stderr: [36mINFO[0m[2022-12-29 15:33:18] reading config file: data/config.json        
stderr: [36mINFO[0m[2022-12-29 15:33:18] load config from env with prefix:            
stderr: [36mINFO[0m[2022-12-29 15:33:18] init logrus...                               
stdout: [31mFATA[0m[2022-12-29 15:33:18] failed to connect database:unable to open database file: no such file or directory 
stderr: [36mINFO[0m[2022-12-29 15:33:20] reading config file: data/config.json

**Comment by NovaShen0724** [comment]
请提供系统环境及架构和alist包名

**Comment by lyqnihao** [comment]
我在用docker版alist时也遇到过同样的错误，运行alist的命令行：docker run -d --restart=always -v /etc/alist:**/opt/alist/data** -p 5244:5244 -e PUID=0 -e PGID=0 -e UMASK=022 --name="alist" xhofe/alist-aria2:latest，查看openwrt的文件系统里，opt目录下没有alist目录，于是修改”/opt/alist/data“到docker根目录下，比如我的docker根目录是”/mnt/mmcblk0p4/docker/“，就把该路径改为”/mnt/mmcblk0p4/docker/alist/data“，然后运行就正常了。不知道这个方法对楼主是否有效

**Reply by jerrybeing** [reply]
感谢分享，我也是这个问题 按你的命令解决了，谢谢。

**Reply by edgexie** [reply]
我有个问题，`/etc/alist:/opt/alist/data` 是指宿主机`/etc/alist`到docker容器的`/opt/alist/data` 吧？
为什么我按照你的方法好了？我的宿主机docker根目录是`/opt/docker`。
`-v /etc/alist:/opt/docker/alist/data`设置就好了，为什么？

**Reply by lyqnihao** [reply]
也许是我前边表述有问题，加粗部分给你造成误解，就是该修改冒号前边宿主机路径的。以我那个例子来说，修改完是“-v /mnt/mmcblk0p4/docker/alist:/opt/alist/data”

**Comment by mapleshadow** [comment]
要把网站TLS证书放入，不然就会报错

**Comment by LYL-0** [comment]
D:\app\alist\alist-windows_3.33.0_amd64>alist server
INFO[2024-07-06 16:10:04] reading config file: data\config.json
FATA[2024-07-06 16:10:04] update config struct error: open data\config.json: Access is denied.
有人知道怎么回事吗
